src/scripts/helpers/api-helper.js

```diff
@@ -0,0 +1,35 @@
+import SessionHelper from '../helpers/session-helper';
+
+export default class APIHelper {
+	constructor() {
+		this.$ = jQuery;
+		this.sessionHelper = new SessionHelper();
+	}
+
+	getCart() {
+		console.log( this.getCartUrl() );
```

Comment:
Geen debugging code in productie. Is er een reden waarom dit niet door de npm workflow wordt afgevangen?

Reply:
fixed https://github.com/level-level/ll-voordemensen/pull/17/commits/75b4c83072d95b9a9bd1dc5251d694b8a4ac48cb

Reply:
stond op off, komt uit clarkson dawn. Misschien toch maar eens aanzetten daar.